control_panel_api/serializers.py

```diff
@@ -266,3 +269,41 @@ class Meta:
             'nickname',
             'name',
         )
+
+
+class ESBucketHitsSerializer(serializers.BaseSerializer):
+
+    def to_representation(self, aggregations):
+        accessed_by_sum_map = {}
+        accessed_by_type_map = {}
+
+        for result in aggregations.bucket_hits:
```

Comment:
I may miss something but looking at the fixture file (`es.py`) shouldn't this be `aggregations.bucket_hits.buckets`?

I mean, test seems to work so maybe the `for ... in` in Python is doing something clever in this case and loop through the `buckets` array implicitly?

Reply:
The elasticsearch library does some overloading of magic methods so it gives you this sometimes unintuitive interface.

Reply:
Magic 🎩